### PR TITLE
update loss & a small change in requirements

### PR DESCRIPTION
**Pull request by FFTYYY**

### CODE REVIEW

File: test/core/test_loss.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+				]
+			)
+
+
+		y = tc.log(y)
+		los = loss_func(y , gy)
+
+		r = -math.log(.3) - math.log(.3) - math.log(.1)
+		r /= 3
+		print ("loss = %f" % (los))
```
Comment (FengZiYjun) [inline test/core/test_loss.py]: Unit tests should use `self.assertXXX` to evaluate the correctness, like `self.assertEqual(los, r)` in this test.
If assertions fail to pass, unit tests find something wrong.

Comment (FengZiYjun) [reply test/core/test_loss.py]: Wonderful.
